### DISCUSSION TITLE
Move ''Always play a slider's tail sample'' from the Classic mod to settings

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#23702] [Ideas] [answered]
I just feel like it would more fit settings since it is a player side thing, enabling the classic mod only for that seems too much.
That option could be in the Gameplay settings right under "Always play first combo break sound" in the Sound section.

**Comment by peppy** [comment, marked as answer]
Things where we think the new behaviour is more correct are intentionally in classic mode. This is one of those cases. If you miss a tail, it's weird that it makes a sound, no?

**Reply by ghost** [reply]
I guess that is true, i'll just keep classic mode on only for that i guess!
